### PR TITLE
feat: add get all dashboards support (New dashboards API)

### PR DESCRIPTION
**Pull request by Morsicus**
## Features

* Implement `get all dashboards` for `/v1/dashboard` (new dashboards API) -> close https://github.com/zorkian/go-datadog-api/issues/253

### CODE REVIEW

File: boards.go [hunk 1]
```diff
@@ -29,6 +29,24 @@ type Board struct {
 	ModifiedAt        *string            `json:"modified_at,omitempty"`
 }
 
+// BoardLite represents a simplify dashboard (without widgets, notify list, ...)
+// It's used when we load all boards.
+type BoardLite struct {
+	Title        *string `json:"title"`
```
Comment (bkabrda) [inline boards.go]: Most often, we prefer putting `omitempty` everywhere to stay consistent, I'd suggest you do that.

File: boards.go [hunk 2]
```diff
@@ -29,6 +29,24 @@ type Board struct {
...
+	Description  *string `json:"description,omitempty"`
+	LayoutType   *string `json:layout_type`
```
Comment (bkabrda) [inline boards.go]: You forgot quotes around `layout_type` (plus also make it `omitempty`, please).

File: boards.go (hunk 1 above)
Comment (Morsicus) [reply boards.go]: amended

File: boards.go (hunk 2 above)
Comment (Morsicus) [reply boards.go]: amended

**Pull request merged**